samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -173,6 +173,8 @@ Emergency data storage
 
 To build this sample with support for emergency data storage, set ``OVERLAY_CONFIG`` to :file:`overlay-emds.conf`.
 This will save replay protection list (RPL) data and some of the :ref:`bt_mesh_lightness_srv_readme` data to the emergency data storage instead of to the :ref:`settings_api`.
+When using EMDS, certain conciderations need to be taken with regards to hardware choices in your application design.
+See :ref:`emds_readme` for more information.
```

Comment:
```suggestion
To build this sample with support for emergency data storage (EMDS), set ``OVERLAY_CONFIG`` to :file:`overlay-emds.conf`.
This will save replay protection list (RPL) data and some of the :ref:`bt_mesh_lightness_srv_readme` data to the emergency data storage instead of to the :ref:`settings_api`.
When using EMDS, certain conciderations need to be taken regarding hardware choices in your application design.
See :ref:`emds_readme` for more information.
```